src/osd/PG.h

```diff
@@ -786,7 +786,9 @@ class PG : public DoutPrefixProvider,
   struct C_DeleteMore : public Context {
     PGRef pg;
     epoch_t epoch;
-    C_DeleteMore(PG *p, epoch_t e) : pg(p), epoch(e) {}
+    int64_t num_objects;
+    C_DeleteMore(PG *p, epoch_t e, int64_t num = 1) : pg(p), epoch(e),
```

Comment:
The default for `num` should be removed here.

Reply:
Yes, I'll make this change. I'll add a comment here to explain: https://github.com/ceph/ceph/blob/main/src/osd/PrimaryLogPG.cc#L12853?

Reply:
Sounds good to me.

Reply:
LGTM